setup.py

```diff
@@ -2,8 +2,7 @@
 import sys
 
 from setuptools import find_packages, setup
-from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
-from wheel.pep425tags import get_platform
+from wheel.bdist_wheel import get_platform, bdist_wheel as _bdist_wheel
```

Comment:
Note: `get_platform` was previously also available in the `bdist_wheel` submodule. So this should still work on older versions of wheel too.

Reply:
I would recommend manually verifying (by locally building) that the wheel has not changed significantly, it's a bit sensitive

Reply:
I checked the source, and we're using the exact same functions, except it's no longer present in the place where we used to import it from.